gemstone/src/block_explorer/explorers/blockchair.rs

```diff
@@ -0,0 +1,54 @@
+use crate::block_explorer::{BlockExplorer, Metadata};
+use primitives::Chain;
+
+static BLOCKCHAIR_NAME: &str = "Blockchair";
+static BLOCKCHAIR_BASE_URL: &str = "https://blockchair.com";
+
+pub struct Blockchair {
+    pub meta: Metadata,
+    pub chain: &'static str,
+}
+
+impl Blockchair {
```

Comment:
more chains for https://blockchair.com/explorers

Reply:
I didn't add all evm chains (only eth/base), you can pick important ones